code/game/objects/structures/electricchair.dm

```diff
@@ -73,6 +73,12 @@
 		M.Stun(60)
 		M.Jitter(60)
 		visible_message("<span class='danger'>The electric chair went off!</span>", "<span class='danger'>You hear a deep sharp shock!</span>")
+		var/mob/living/carbon/human/H = null
+		if(ishuman(M))
+			H = M
+			if(isgolem(H)) //golems can't be burned
```

Comment:
Wouldn't it be better and less snowflakey to just handle golems in proc/burn_skin()?